syntax = "proto3";

package match_service;

service MatchService {

    rpc CreateMatch (CreateMatchRequest) returns (CreateMatchReply);

}

message CreateMatchRequest {
    string sender_id = 1;
    string receiver_id = 2;
    int32 status = 3;
}

message CreateMatchReply {
    string status = 1;
}